src/applications/check-in/day-of/pages/CheckIn/index.jsx

```diff
@@ -1,20 +1,23 @@
-import React, { useCallback, useMemo } from 'react';
+import React, { useCallback, useEffect, useState, useMemo } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 import { api } from '../../../api';
 import { useFormRouting } from '../../../hooks/useFormRouting';
 import { receivedMultipleAppointmentDetails } from '../../../actions/day-of';
 
 import DisplayMultipleAppointments from './DisplayMultipleAppointments';
+import { URLS } from '../../../utils/navigation';
 
 import {
   makeSelectVeteranData,
   makeSelectCurrentContext,
+  makeSelectForm,
 } from '../../../selectors';
 
 const CheckIn = props => {
+  const { isDayOfDemographicsFlagsEnabled } = props;
```

Comment:
We have a selector for the feature toggles so we don’t need to rely on props to get the value of the feature toggles,

Reply:
Thanks! This branch has been updated to use the selectors instead 

commits [ce1fbcd](https://github.com/department-of-veterans-affairs/vets-website/pull/20339/commits/ce1fbcdcf9fc3b4fbe6d8dc2fdfad1239e988c16) and [b7960b3](https://github.com/department-of-veterans-affairs/vets-website/pull/20339/commits/b7960b3ea532f95bc1b77468854031e165343db8)